src/app/pokemon/components/pokemons-modal-vs/pokemons-modal-vs.component.html

```diff
@@ -0,0 +1,185 @@
+<mat-dialog-content class='mat-typography setPaddingModal'>
+  <mat-grid-list
+    class='setMarginModal'
+    mat-dialog-title
+    cols='12'
+    rowHeight='1.5rem'
+  >
+    <mat-grid-tile colspan='12' rowspan='2'>
+      <span class='fix-text'
+        >{{ selectedPokemons[0].name }} VS {{ pokemonName2 }}</span
+      >
+    </mat-grid-tile>
+  </mat-grid-list>
+  <hr class='setMarginModalLines' />
+  <mat-grid-list cols='12' rowHeight='4rem'>
+    <mat-grid-tile colspan='6' rowspan='2'>
+      <img
+        mat-card-image
+        class='imgFluid'
+        src='{{ image }}'
+        alt=' Not found image'
+      />
+    </mat-grid-tile>
+    <mat-grid-tile colspan='6' rowspan='2'>
+      <img
+        mat-card-image
+        class='imgFluid'
+        src='{{ image2 }}'
+        alt=' Not found image'
+      />
+    </mat-grid-tile>
+  </mat-grid-list>
+  <hr class='setMarginModalLines' />
+  <mat-grid-list cols='12' rowHeight='2rem'>
+    <mat-grid-tile colspan='5' rowspan='1'>
+      <div mat-dialog-content>
+        <table style='width: 100%'>
+          <tr>
+            <span class='text-description'>
+              <td>{{ descriptionPokemons[0].height / 10 }}m</td>
```

Comment:
don't use interpolation to make calculations, they are run every render

Reply:
I replaced them with pipes